DS and Algorithm/Selenium/SeleniumExample.java

```diff
@@ -0,0 +1,23 @@
+// src/main/java/SeleniumExample.java
+public class SeleniumExample {
+    public static void main(String[] args) {
+        // Initialize WebDriver
+        WebDriver driver = WebDriverSetup.initializeChromeDriver();
```

Comment:
**Issue**: The WebDriver is initialized without checking if the driver executable exists or if it's compatible with the installed browser version. <br> **Fix**: Validate the existence and version compatibility of the chromedriver executable before initializing WebDriver. <br> **Code Suggestion**: 
 ```
 + File file = new File("lib/chromedriver.exe");
 + if(!file.exists()) throw new FileNotFoundException("ChromeDriver executable not found at " + file.getPath());
 + // Existing code for WebDriver initialization
 ```